src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+
+const message = document.createElement('div');
+const errorMessage = document.createElement('div');
+
+message.classList = 'message';
+message.innerText = 'Promise was resolved!';
+
+errorMessage.classList = 'message error-message';
+errorMessage.innerText = 'Promise was rejected!';
+
+function createPromise() {
+  const resolver = (resolve, reject) => {
+    logo.addEventListener('click', () => {
+      resolve();
+    });
+
+    setTimeout(() => {
+      reject();
+    }, 3000);
+  };
+
+  return new Promise(resolver);
+}
+
+const promice1 = createPromise();
```

Comment:
Typo in variable names: 'promice1' and 'promice2' should be 'promise1' and 'promise2'.